src/Makefile

```diff
@@ -438,7 +449,18 @@ DEP = $(REDIS_SERVER_OBJ:%.o=%.d) $(REDIS_CLI_OBJ:%.o=%.d) $(REDIS_BENCHMARK_OBJ
 # The following files are checked in and don't normally need to be rebuilt. They
 # are built only if python is available and their prereqs are modified.
 ifneq (,$(PYTHON))
-$(COMMANDS_DEF_FILENAME).def: commands/*.json ../utils/generate-command-code.py
+FORCE:
```

Comment:
I believe the .PHONY is a more commonly used paradigm, is there a reason you are doing it this way?

Reply:
.PHONY is for when you need a target that will never exist and forces other targets to run since they are dependencies to the .PHONY. This is different. Here I have a generated file as the target but even if it exists, I want the target to run. If I call it .PHONY it creates a loop in the dependencies (I just tried it again).